docs/references/http-websocket-apis/index.md

```diff
@@ -6,7 +6,60 @@ metadata:
 ---
 # HTTP / WebSocket APIs
 
-Communicate directly with rippled, the core peer-to-peer server that manages the XRP Ledger.
+You can communicate with the XRP Ledger through the `rippled` servers' publicly available APIs.
 
+Currently, there are two API versions: `1` and `2` {% badge href="https://github.com/XRPLF/rippled/releases/tag/2.0.0" %}New in: rippled 2.0.0{% /badge %}. The server reports the range of supported API versions in the `version` API method. <!-- STYLE_OVERRIDE: will --> <!-- TODO: add a link when `version` method is documented. -->
+
+Separate API requests can use different API versions even on the same persistent connection. For example, if you connect through WebSocket to a server that supports API versions 1 and 2, you can make an `account_tx` request using API version 2 and then make another `account_tx` request using API version 1 from the same connection.
```

Comment:
Would be helpful to clarify a few things:
- The default API versions used when requests are made through websockets (vs others like SDKs).
- One can specify the API version you want to use for a specific request.

Reply:
I moved the table up so the info about default API version used is more obvious. Also added a line about specifying version.